.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Cool, so I did
`yarn add -D eslint` and
`eslint --init` 

Then updated my .eslint.rc as suggested. I'm getting this error:
` [Info  - 14:30:40] ESLint server is running.
[Info  - 14:30:41] ESLint library loaded from: /Users/markevans/node_modules/eslint/lib/api.js
[Error - 14:30:47] Cannot find module 'eslint-config-airbnb' Referenced from: /Users/markevans/Workspace/courses/ReactNative/stripe/.eslintrc.js
`